pkg/dockerregistry/server/auth.go

```diff
@@ -45,6 +45,8 @@ const (
 
 	RealmKey      = "realm"
 	TokenRealmKey = "tokenrealm"
+
+	accessControllerOptionContext = "_context"
```

Comment:
Make it public and use it in `pkg/cmd/dockerregistry/dockerregistry.go`
Godoc please :)